ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -86,32 +100,35 @@ def _llmobs_set_metadata(self, span: Span, model_provider: Optional[str] = None)
             span.set_tag_str(METADATA, json.dumps(metadata))
 
     def _llmobs_set_meta_tags_from_llm(
-        self, span: Span, prompts: List[Any], completions: Any, err: bool = False
+        self, span: Span, prompts: List[Any], completions: Any, err: bool = False, is_workflow=False
     ) -> None:
-        span.set_tag_str(SPAN_KIND, "llm")
+        span.set_tag_str(SPAN_KIND, "workflow" if is_workflow else "llm")
         span.set_tag_str(MODEL_NAME, span.get_tag(MODEL) or "")
         span.set_tag_str(MODEL_PROVIDER, span.get_tag(PROVIDER) or "")
 
+        input_tag_key = INPUT_VALUE if is_workflow else INPUT_MESSAGES
+        output_tag_key = OUTPUT_VALUE if is_workflow else OUTPUT_MESSAGES
+
         if isinstance(prompts, str):
             prompts = [prompts]
-        span.set_tag_str(INPUT_MESSAGES, json.dumps([{"content": str(prompt)} for prompt in prompts]))
+
+        span.set_tag_str(input_tag_key, json.dumps([{"content": str(prompt)} for prompt in prompts]))
 
         message_content = [{"content": ""}]
         if not err:
             message_content = [{"content": completion[0].text} for completion in completions.generations]
-        span.set_tag_str(OUTPUT_MESSAGES, json.dumps(message_content))
+        span.set_tag_str(output_tag_key, json.dumps(message_content))
 
     def _llmobs_set_meta_tags_from_chat_model(
-        self,
-        span: Span,
-        chat_messages: List[List[Any]],
-        chat_completions: Any,
-        err: bool = False,
+        self, span: Span, chat_messages: List[List[Any]], chat_completions: Any, err: bool = False, is_workflow=False
```

Comment:
```suggestion
        self, span: Span, chat_messages: List[List[Any]], chat_completions: Any, err: bool = False, is_workflow: bool=False
```
(just for consistency), although that might put you back into multi-line length

Reply:
Thanks for catching this!